test/libsolidity/SolidityOptimizer.cpp

```diff
@@ -1296,7 +1296,7 @@ BOOST_AUTO_TEST_CASE(constant_optimization_early_exit)
 			// Store and hash
 			assembly {
 				mstore(32, x)
-				ret := sha3(0, 40)
+				ret := keccak256(0, 40)
```

Comment:
Please also add a test that shows that `sha3` is still available in inline assembly.